src/components/App.js

```diff
@@ -30,15 +30,15 @@ class App extends Component {
 
   render() {
     const { movies, errorMessage } = this.props
-    
+
     return (
       <Router>
-        <div>
-          <h1>Movie Tracker</h1>
-          <header>
+        <div className="App">
+          <header className="App-header">
             <Nav />
+            <img src={logo} alt="Logo" className="App-img"/>
           </header>
-          <UserForm />
```

Comment:
I believe when we add this back in it will need to go above the MoviesContainer.